rubygem-azure-core/rubygem-azure-core.spec

```diff
@@ -0,0 +1,89 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Generated from azure-0.7.5.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name azure
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.7.5
+Release: 1%{?dist}
+Summary: Official ruby client library to consume Microsoft Azure services
+Group: Development/Languages
+License: Apache License, Version 2.0
+URL: http://github.com/azure/azure-sdk-for-ruby
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix}rubygem(faraday) >= 0.9
+Requires: %{?scl_prefix}rubygem(faraday) < 1.0
+Requires: %{?scl_prefix}rubygem(faraday_middleware) >= 0.10
+Requires: %{?scl_prefix}rubygem(faraday_middleware) < 1.0
+Requires: %{?scl_prefix_ror}rubygem(nokogiri) >= 1.6
+Requires: %{?scl_prefix_ror}rubygem(nokogiri) < 2.0
+BuildArch: noarch
+
+%description
+Microsoft Azure Client Library for Ruby.
```

Comment:
"This project provides a Ruby package with core functionality consumed by Azure SDK gems."